app/Http/Controllers/Legacy/Web/InboxController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Rogue\Http\Controllers\Legacy\Web;
+
+use Rogue\Services\CampaignService;
+use Rogue\Http\Controllers\Controller;
+
+class InboxController extends Controller
```

Comment:
Was the stuff in this file pulled out of `AshesCampaignsController`?

Reply:
Yes! This was originally the `showInbox` method in `AshesCampaignsController`.